db/migrate/20221129094342_create_value_definitions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Data model for storing value definitions
+class CreateValueDefinitions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :value_definitions, id: :uuid do |t|
+      t.string :ref_id
+      t.string :title
+      t.text :description
+      t.string :value_type
+      t.string :default_value
+      t.numeric :lower_bound, default: nil
+      t.numeric :upper_bound, default: nil
+      t.uuid :benchmark_id
+      t.references :benchmark_id, type: :uuid, index: true
+    end
+
+    add_column :rules, :value_checks, :uuid, array: true
+    add_column :profiles, :value_overrides, :json
```

Comment:
@marleystipich2 I'm not sure about the naming here. If we do the refinements/selectors, would we merge that information here into the column statically? In this case the naming is correct...otherwise we have the generic problem of starting a metal band.

Reply:
I think `value_overrides` would be okay for profile values that were refined but doesn't make sense for profile values that weren't refined because nothing was overwritten.

But I don't know if we need to track values for profiles that haven't been refined  anyway because if a value isn't in `value_overrides` then we'll know that the profile didn't refine it so they can get the default value from the `values` table?

Reply:
Talked about this over slack and decided `value_overrides` should be good 👍 Updated the architecture doc with more info about this column and what it will be used for